command/issue.go

```diff
@@ -29,14 +29,14 @@ func init() {
 	issueCreateCmd.Flags().StringP("body", "b", "",
 		"Supply a body. Will prompt for one otherwise.")
 	issueCreateCmd.Flags().BoolP("web", "w", false, "Open the browser to create an issue")
-	issueCreateCmd.Flags().StringSliceP("assignee", "a", nil, "Assign a person by their `login`")
-	issueCreateCmd.Flags().StringSliceP("label", "l", nil, "Add a label by `name`")
-	issueCreateCmd.Flags().StringSliceP("project", "p", nil, "Add the issue to a project by `name`")
+	issueCreateCmd.Flags().StringSliceP("assignees", "a", nil, "Assign people by their `login`")
```

Comment:
Let's avoid renaming any of these flags primarily because this breaks backwards compatibility, and we tend to avoid doing that even in beta (unless there is a strong reason to do so). Please undo all of these renames.

To communicate to people that they are able to pass multiple values for these flags we should expand our command docs instead, but let's do that in a separate PR.

Reply:
Cool, I will do that in a separate PR